api/v1/models/Hacker.js

```diff
@@ -0,0 +1,43 @@
+/* jshint esversion: 6 */
+
+var registration = require('../utils/registration');
+var _ = require('lodash');
+
+var OMITTED_ATTRIBUTES = [];
+
+var Model = require('./Model');
+var Hacker = Model.extend({
+	tableName: 'hackers',
+	idAttribute: 'id',
+	hasTimestamps: ['created', 'updated'],
+	validations: {
+		userId: ['required', 'integer'],
+		age: ['number', 'max:100', 'min:13'],
+		gender: ['string', registration.verifyGender],
+		school: ['string', 'maxLength:255'],
+		graduationYear: ['number', 'between:2016:2021'],
+		major: ['string', 'maxLength:255'],
+		diet: ['string', registration.verifyDiet],
```

Comment:
For now, it's not required because I was under the impression that we'd like to allow hackers to partially fill out their registration and save it for later. If things are required, we can't save partial registrations.